javascript-carousel/main.js

```diff
@@ -0,0 +1,115 @@
+var $img = document.querySelector('img');
+var imagesArr = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+var count = 1; // counts start at 1 since the default html image is index 0
+var $circularButtons = document.querySelectorAll('.dot');
+var circleCounter = 0;
```

Comment:
Try to rewrite your code to only keep track of the `index` of the active image. The dot element and image url index are the same number.